apps/omg_watcher/lib/omg_watcher/web/controllers/transaction.ex

```diff
@@ -63,4 +64,22 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
       |> api_response(conn, :create)
     end
   end
+
+  defp increment_metrics_counter(response) do
+    case response do
+      {:error, {:validation_error, _, _}} ->
+        Appsignal.increment_counter("transaction.failed.validation", 1)
+
+      {:error, {error, _}} when is_atom(error) ->
+        Appsignal.increment_counter("transaction.fail." <> Atom.to_string(error), 1)
```

Comment:
are you sure this works?
also from Appsignal docs:
```
We recommend naming your metrics something easily recognizable and without too many dynamic elements. While you can wildcard parts of the metric name for dashboard creation, we recommend you only use this for small grouping and not use IDs for metric names.
```

Reply:
there are three error `:client_error`, `:server_error`,`:malformed_response` <-- answer for Appsignal it's small group for error.
Question is if we don't want this in a different way (is there `one_of_atoms` like funkcion?)